molecule/default/prepare.yml

```diff
@@ -14,10 +14,17 @@
     - name: install aptitude for ansible
       apt: name=aptitude state=present cache_valid_time=3600
 
+    - name: Create operators group
+      group: name=operators state=present
```

Comment:
Adding this to the `prepare.yml` means we're not testing the role in the "new install" case. I would create a new molecule scenario (`molecule init scenario`) to test the inactive+removed user scenario separate from this.